routes.js

```diff
@@ -0,0 +1,82 @@
+var Todo = require('./models/todo')(Todo);
+
+module.exports = (server) => {
+    //Test 1: Create a Todo
+    server.post('/todos', async (req, res, next) => {
+        try {
+            let objects = await Todo.create({
+                title: req.body.title,
+                completed: false
+            });
+            res.status(201);
+            res.charSet('utf-8');
+            res.send(objects.dataValues);            
```

Comment:
Don't use `dataValues`, construct the JSON yourself so it doesnt included the dates